engine.py

```diff
@@ -334,6 +333,7 @@ def post_app_init_hiero(self, menu_name="Shotgun"):
             # (several of the key scene callbacks are in the main init file).
             import tk_nuke
             import hiero
+            from hiero.core import env as hiero_env
```

Comment:
Full code of the orignal method if [here](https://github.com/shotgunsoftware/tk-nuke/blob/master/engine.py#L334-L364) This import was missing, which turned out to throw an error everytime we tried to log usage stats, which itself is exception safe so it silently failed for bad reasons all the time.